esi_leap/common/policy.py

```diff
@@ -75,17 +88,21 @@
     policy.DocumentedRuleDefault(
         'esi_leap:offer:get',
         'rule:is_admin or rule:is_owner or rule:is_lessee',
-        'Retrieve offer',
-        [{'path': '/offers', 'method': 'GET'},
-         {'path': '/offers/{offer_ident}', 'method': 'GET'}]),
+        'Retrieve a single offer',
+        [{'path': '/offers/{offer_ident}', 'method': 'GET'}]),
+    policy.DocumentedRuleDefault(
+        'esi_leap:offer:get_all',
+        'rule:is_admin or rule:is_owner or rule:is_lessee',
+        'Retrieve multiple offers',
+        [{'path': '/offers', 'method': 'GET'}]),
     policy.DocumentedRuleDefault(
         'esi_leap:offer:delete',
-        'rule:is_admin or rule:is_owner',
+        'rule:is_admin or rule:is_offer_owner',
```

Comment:
Was the previous behavior really that any owner could delete any offer? That sounds... kind of bad lol

Reply:
Kind of! Previously, the policy check for delete would pass as long as you had the owner role; then an additional check in the controller would verify that you were the owner of that offer. This change gets rid of the need for that second check.